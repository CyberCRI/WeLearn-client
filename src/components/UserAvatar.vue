<template>
  <div class="avatar">
    <img alt="chat user avatar" src="@/assets/user.png" />
  </div>
</template>

<style scoped>
@media screen and (min-width: 576px) {
}
</style>
