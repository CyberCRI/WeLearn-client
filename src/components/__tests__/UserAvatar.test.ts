import { describe, it, expect } from 'vitest';

import { mount } from '@vue/test-utils';
import UserAvatar from '../UserAvatar.vue';

describe('UserAvatar', () => {
  it('renders properly', () => {
    const wrapper = mount(UserAvatar);
    expect(wrapper.element).toMatchSnapshot();
  });
});
