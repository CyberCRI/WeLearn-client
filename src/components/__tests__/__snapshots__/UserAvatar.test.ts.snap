// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UserAvatar > renders properly 1`] = `
<div
  class="avatar"
  data-v-9b599479=""
>
  <img
    alt="chat user avatar"
    data-v-9b599479=""
    src="/src/assets/user.png"
  />
</div>
`;
